presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -159,78 +159,78 @@ public void doesNotFire()
-          fail(String.format(
-                  "%s did not fire for:\n%s",
-                  rule.getClass().getName(),
-                  formatPlan(plan, types, ruleApplication.lookup)));
-      }
-
-      PlanNode actual = ruleApplication.getResult();
-
-      if (actual == plan) { // plans are not comparable, so we can only ensure they are not the same instance
-          fail(String.format(
-                  "%s: rule fired but return the original plan:\n%s",
-                  rule.getClass().getName(),
-                  formatPlan(plan, types, ruleApplication.lookup)));
-      }
-
-      if (!ImmutableSet.copyOf(plan.getOutputSymbols()).equals(ImmutableSet.copyOf(actual.getOutputSymbols()))) {
-          fail(String.format(
-                  "%s: output schema of transformed and original plans are not equivalent\n" +
-                          "\texpected: %s\n" +
-                          "\tactual:   %s",
-                  rule.getClass().getName(),
-                  plan.getOutputSymbols(),
-                  actual.getOutputSymbols()));
-      }
-
-      inTransaction(session -> {
-          assertPlan(session, metadata, ruleApplication.lookup, new Plan(actual, types, ruleApplication.lookup, session), pattern);
-          return null;
-      });
-  }
-
-  private RuleApplication applyRule()
-  {
-      SymbolAllocator symbolAllocator = new SymbolAllocator(symbols);
-      Memo memo = new Memo(idAllocator, plan);
-
-      if (!rule.getPattern().matches(plan)) {
-          return new RuleApplication(lookup, symbolAllocator.getTypes(), Optional.empty());
-      }
-
-      Optional<PlanNode> result = inTransaction(session -> rule.apply(memo.getNode(memo.getRootGroup()), lookup, idAllocator, symbolAllocator, session));
-
-      return new RuleApplication(lookup, symbolAllocator.getTypes(), result);
-  }
-
-  private String formatPlan(PlanNode plan, Map<Symbol, Type> types, Lookup lookup)
-  {
-      return inTransaction(session -> PlanPrinter.textLogicalPlan(plan, types, metadata, lookup, session, 2));
-  }
-
-  private <T> T inTransaction(Function<Session, T> transactionSessionConsumer)
-  {
-      return transaction(transactionManager, accessControl)
-              .singleStatement()
-              .execute(session, session -> {
-                  // metadata.getCatalogHandle() registers the catalog for the transaction
-                  session.getCatalog().ifPresent(catalog -> metadata.getCatalogHandle(session, catalog));
-                  return transactionSessionConsumer.apply(session);
-              });
-  }
-
-  private static class RuleApplication
-  {
-      private final Lookup lookup;
-      private final Map<Symbol, Type> types;
-      private final Optional<PlanNode> result;
-
-      public RuleApplication(Lookup lookup, Map<Symbol, Type> types, Optional<PlanNode> result)
+        RuleApplication ruleApplication = applyRule();
+        Map<Symbol, Type> types = ruleApplication.types;
```

Comment:
You can extract formatting fixes to a separate PR and ask @kokosing to merge it faster.